decidim-initiatives/app/controllers/decidim/initiatives/widgets_controller.rb

```diff
@@ -15,7 +15,7 @@ class WidgetsController < Decidim::WidgetsController
       private
 
       def model
-        @model ||= current_initiative
+        @model ||= current_initiative if current_initiative.published?
```

Comment:
I was looking to this statement, and i was wondering why we do not in account the `moderated` state. I see that an initiative can be created by an user, but there is no method to moderate it, and an admin cannot unpublish it.

Reply:
Initiatives do not have moderations. Saying that, this suggestion was taken into account in the following commits: 

- Check for moderation in the resources that have that feature: 514689182db2fead7961efc6af094d961cb91d1d
- Check for the different initiatives' states: 1a4a553